docs/tools/packman/bootstrap/install_package.py

```diff
@@ -86,7 +86,7 @@ def __exit__(self, type, value, traceback):
 def rename_folder(staging_dir: StagingDirectory, folder_name: str):
     try:
         staging_dir.promote_and_rename(folder_name)
-    except OSError as exc:

```

Comment:
We shouldn't be formatting anything in `packman` because this is vendored code.

Reply:
If it is helpful, this is how we exclude files in RAPIDS: [pyproject.toml](https://github.com/rapidsai/cudf/blob/cd3a79bfa71be68c8e95ff8dd60a41eb641f8d5a/pyproject.toml#L118-L121)

Reply:
Will exclude (currently testing here: https://github.com/NVIDIA/cccl/pull/3130/commits/8ac225484c96964e864644a587a0eb53979a7d01
)